docs/tutorials/use-dicom-web-standard-apis-with-curl.md

```diff
@@ -80,7 +81,7 @@ _Details:_
  > * `Content-Type: multipart/related; boundary=ABCD1234`
  > * `Content-Type: multipart/related`
 
-`curl --request POST "http://{service-name}.azurewebsites.net/studies/1.2.826.0.1.3680043.8.498.13230779778012324449356534479549187420" --header "Accept: application/dicom+json" --header "Content-Type: multipart/related; type=\"application/dicom\"" --form "file1=@{path-to-dicoms}/blue-circle.dcm;type=application/dicom"`
+`curl --request POST "http://{service-name}.azurewebsites.net/v{version}/studies/1.2.826.0.1.3680043.8.498.13230779778012324449356534479549187420" --header "Accept: application/dicom+json" --header "Content-Type: multipart/related; type=\"application/dicom\"" --form "file1=@{path-to-dicoms}/blue-circle.dcm;type=application/dicom"`

```

Comment:
Is there a simpler way to specify the base url in one place

Like start the doc with base url like
{Web service url}/{version}

and reference the base url everywhere?

Reply:
Updated, thanks!